src/pages/zh-CN/comparing-astro-vs-other-tools.md

```diff
@@ -0,0 +1,288 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Astro vs. X
+description: 比较 Astro 和诸如 Gatsby、Next.js、Nuxt、Hugo、Eleventy 之类的静态网站生成器。
+---
+<!-- TODO: UNcomment out the parts re: number of bytes of JS etc, once we decide which values/markers we'd like to use here. -->
+我们经常会被问道这样一个问题“Astro 和我最喜欢的 **\_\_\_\_** 项目比起来怎么样？”
+
+本指南就是为了回答这个问题而编写的，适用于几个流行的网站生成器和 Astro 替代品。
+
+两个重要特性使得 Astro 与大多数替代品不同。
+
+- [按需加载](/en/core-concepts/partial-hydration)
```

Comment:
```suggestion
- [极简耦合](/zh-CN/core-concepts/partial-hydration/)
```

Reply:
What meaning of `偏`? `按需加载` is used here instead of `部分激活` because I got a better view of the translation of the word halfway through the translation of the document. Here's the part I accidentally left out.